_dashboards/maps.md

```diff
@@ -0,0 +1,154 @@
+
+OpenSearch supports displaying documents that contain both geopoint and geoshape fields on a map.
+{: .note}
+
+The following example assumes that you have the `opensearch_dashboards_sample_data_flights` dataset installed. If you don't have this dataset installed, perform the following steps:
+
+1. On the top menu bar, go to **OpenSearch Dashboards > Overview**.
+1. Select the **View app directory** link. 
+1. Select **Add sample data**.
+1. In the **Sample flight data** panel, select the **Add data** button .
+
+Add a document layer as follows:
+
+1. In the **Layers** panel, select the **Add layer** button.
+1. From the **Add layer** dialog, select **Data layer > Documents**.
+1. In **Data source**, select `opensearch_dashboards_sample_data_flights`. Alternatively, you can enter another index pattern to visualize.
+1. In **Geospatial field**, select a geospatial field (geopoint or geoshape) to be displayed in the visualization. In this example, select `DestLocation`.
+1. (Optional) Select the **Style** tab to change the fill color, border color, border thickness, or marker size.
+1. Select the **Settings** tab to edit layer settings.
+1. Enter `Flight destination` in **Name**.
+1. Select the **Update** button.
+1. To see more data, in the upper-right corner select the calendar icon dropdown, then under **Quick select** choose **Last 15 days**. 
+
+You should see the flight destination data similar to the following image:
+
+<img src="{{site.url}}{{site.baseurl}}/images/maps/new-layer.png" alt="Flight destination map">
+
+## Filtering data
+
+To show a subset of the data in the index, filter the data. The following example filters the Flight destination data to see only United States destinations:
+
+1. In the **Layers** panel, select **Flight destination**.
+1. Select **Filters**.
+1. Select **Add filter**.
+1. In **Edit filter**, select **DestCountry** in **Field**.
+1. In **Operator**, select **is**.
+1. In **Value**, select **US**.
+1. Select the **Save** button.
+1. Select the **Update** button.
+
+For large datasets, you may want to avoid loading data for the whole map. To load data only for the part of the map that is currently visible, select the **Only request data around map extent** checkbox.
+{: .tip}
+
+## Using tooltips to visualize additional data
+
+Document layers show geopoint and geoshape document fields as locations on the map. To add more information to the locations, you can use tooltips. For example, you may want to to show flight delay, destination weather, and destination country in the Flight destination layer. Perform the following steps to configure tooltips to show additional data:
+
+1. In the **Layers** panel, select **Flight destination**.
+1. Select **Tooltips**. 
+1. Select the **Show tooltips** checkbox.
+1. In the **Tooltip fields** dropdown, select the fields that you'd like to display. In this example, select `FlightDelay`, `DestWeather`, and `DestCountry`.
```

Comment:
Change "dropdown" to "dropdown list" (per [TERMS.md](https://github.com/opensearch-project/documentation-website/blob/main/TERMS.md))